app/package.json

```diff
@@ -27,18 +27,22 @@
     "d3": "^7.3.0",
     "datavoyager": "^2.0.0-alpha.24",
     "jsonschema": "^1.4.0",
+    "jszip": "^3.7.1",
     "rdf-literal": "^1.1.0",
     "register-service-worker": "^1.7.1",
     "smiles-drawer": "^1.2.0",
+    "socket.io-client": "^4.4.1",
     "v-jsoneditor": "^1.4.5",
     "vega": "^5.21.0",
     "vega-embed": "^6.20.5",
     "vega-lite": "^5.2.0",
     "vega-lite4": "npm:vega-lite@^4.17.0",
     "vega-lite5": "npm:vega-lite@^5.1.1",
     "vue": "^2.6.11",
+    "vue-advanced-cropper": "^0.16.8",
```

Comment:
This version of the library is outdated (a long time ago) and it contains different issues.